base_cron_exclusion/models/__init__.py

```diff
@@ -1,4 +1,4 @@
-# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
```

Comment:
Please do not add copyrights to init files. See template https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py (there are also a discussion on maintainer-tools closed PRs if you are interested)

Reply:
Done.